app/views/users/webauthn_setup/new.html.erb

```diff
@@ -1,14 +1,16 @@
+<% setup_context = @platform_authenticator ? 'webauthn_platform_setup' : 'webauthn_setup' %>
+
 <% title t('titles.totp_setup.new') %>
 <% help_link = new_window_link_to t('links.what_is_webauthn'),
                                   MarketingSite.help_hardware_security_key_url %>
 
 <%= image_tag asset_url('security-key.svg'), alt: '', width: '90', class: 'margin-left-1' %>
 
 <h1 class='margin-y-0'>
-  <%= t('headings.webauthn_setup.new') %>
+  <%= t("headings.#{setup_context}.new") %>
 </h1>
 <p class='mt-tiny margin-bottom-4'>
-  <%= t('forms.webauthn_setup.intro_html', link: help_link) %>
+  <%= t("forms.#{setup_context}.intro_html", link: help_link) %>
```

Comment:
```suggestion
  <%= t('forms.#{setup_context}.intro_html', link: help_link) %>
```

Reply:
For ruby, you have to use double quoted strings for interpolation to work. Single quoted strings do not have escape characters or interpolations